### ISSUE TITLE
Installation with Node 12 fails

### ISSUE THREAD

**Issue by ribeaud** [astefanutti/decktape#184]
Hi,
Installation via `npm install -g decktape` did NOT work on my side. I am getting following build error:
```
node-pre-gyp ERR! stack Error: Failed to execute '/usr/local/Cellar/node/12.3.1/bin/node /usr/local/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js build --fallback-to-build --module=/usr/local/lib/node_modules/decktape/node_modules/hummus/binding/hummus.node --module_name=hummus --module_path=/usr/local/lib/node_modules/decktape/node_modules/hummus/binding --napi_version=4 --node_abi_napi=napi' (1)
node-pre-gyp ERR! stack     at ChildProcess.<anonymous> (/usr/local/lib/node_modules/decktape/node_modules/hummus/node_modules/node-pre-gyp/lib/util/compile.js:83:29)
node-pre-gyp ERR! stack     at ChildProcess.emit (events.js:200:13)
node-pre-gyp ERR! stack     at maybeClose (internal/child_process.js:1021:16)
node-pre-gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:283:5)
node-pre-gyp ERR! System Darwin 18.6.0
node-pre-gyp ERR! command "/usr/local/Cellar/node/12.3.1/bin/node" "/usr/local/lib/node_modules/decktape/node_modules/hummus/node_modules/.bin/node-pre-gyp" "install" "--fallback-to-build"
node-pre-gyp ERR! cwd /usr/local/lib/node_modules/decktape/node_modules/hummus
node-pre-gyp ERR! node -v v12.3.1
node-pre-gyp ERR! node-pre-gyp -v v0.10.3
node-pre-gyp ERR! not ok
```
Is my **Node** version too recent? Attaching the whole build log:

[2019-05-29T05_05_16_677Z-debug.log](https://github.com/astefanutti/decktape/files/3230853/2019-05-29T05_05_16_677Z-debug.log)

Kindest regards. Thanks and cheers,

christian

**Comment by astefanutti** [comment]
It is possible Node 12 is not yet supported by the versions of `node-gyp` / `node-pre-gyp` used. Could you try upgrading `node-gyp` to the latest version?

I'll test it ASAP on my side.

**Comment by ribeaud** [comment]
Mmmmh... But how can I enforce **decktape** installation to use that version? This is what I did so far:
1. `npm install -g node-gyp`
1.  `npm list -g | grep node-gyp` returns the following:
```
├─┬ node-gyp@4.0.0
│ ├─┬ node-gyp@3.8.0
│ │ ├── node-gyp@3.8.0 deduped
```
**Decktape** installation still failing. _--force_ did not help (hoping that the cache will NOT be used for installation).

Any hint? Is there something more I could do to help?

Cheers,

christian

**Comment by astefanutti** [comment]
Could you try installing from the `master` branch, with:

    $ npm install -g "astefanutti/decktape#master"

Version 2.9.2 depends on HummusJS that is not compatible with Node v12. It should be fixed in `master`.

If you confirm it works, I'll cut a new release.

**Comment by ribeaud** [comment]
OK, I made the installation following your instructions. And I did a test with:
```
decktape remark https://ribeaud.github.io/WebEngineering/lectures/11 11.pdf
```
It worked like a charm... 😏

Thanks a lot and have a beautiful day.
Best regards,

christian

**Comment by astefanutti** [comment]
Thanks for the quick feedback. I've just released version 2.9.3.
